app/src/main/java/com/iti/android/team1/ebuy/model/datasource/repository/Repository.kt

```diff
@@ -133,6 +133,11 @@ class Repository(
 
     override fun setAuthStateToPrefs(state: Boolean) = localSource.setAuthStateToPrefs(state)
 
+    override fun logOut() {
+        localSource.setAuthStateToPrefs(false)
+        localSource.setUserIdToPrefs("")
+    }
+
```

Comment:
remove FavoriteId and CartId from shared pref and you can use scope function (apply)

Reply:
done